scripter/scripter.go

```diff
@@ -153,4 +173,113 @@ func SetBasicMethods(c ScrConn, service string) {
 			log.Warning(message)
 		}
 	}, service)
+
+	c.SetStringFunction("getRequest", func() string {
```

Comment:
Isn't this mostly a function only needed for the generic scripter? Adding them globally here looks a bit redundant.